tensorflow/python/kernel_tests/math_ops/cwise_ops_binary_test.py

```diff
@@ -260,6 +264,16 @@ def testFloatVariableOverload(self):
     self.assertAllEqual(np_result, left_result)
     self.assertAllEqual(np_result, right_result)
 
+  @test_util.run_deprecated_v1
```

Comment:
Is it possible to remove this?  We want all new tests to be v2.

Reply:
Done!